webhooks-with-pgstream.mdx

```diff
@@ -0,0 +1,555 @@
+
+In this article we're going to dive in the pgstream webhooks functionality, including more advanced topics, like sending the old and new values, TOAST support, and schema changes support.
+
+![Postgres webhooks with pgstream](/images/pgstream-webhooks.png)
+
+## Preparation
+
+For this tutorial we need a Postgres instance that has logical replication enabled and the [wal2json](https://github.com/eulerto/wal2json) output plugin loaded. There are a number of ways to do this, but we're going to use the [docker-compose file](https://github.com/xataio/pgstream/blob/main/build/docker/docker-compose.yml) from the pgstream repo:
+
+```sh
+git clone https://github.com/xataio/pgstream.git
+```
+
+And then start the Postgres instance:
+
+```sh
+docker compose -f build/docker/docker-compose.yml up db
+```
+
+Then, in a new terminal window, you can connect to it and create a simple table to play with:
+
+```sh
+psql "postgres://postgres:postgres@localhost:5432/postgres"
+```
+
+At the psql prompt:
+
+```sql
+CREATE TABLE users(id SERIAL PRIMARY KEY, name TEXT, role TEXT);
+```
+
+Let's now install `pgstream`. One option is to build it from source, since we already have the repo cloned:
+
+```sh
+go build
+```
+
+The above assumes you have Go installed. If you don't, it might be easier to use the Homebrew tap:
+
+```sh
+brew tap xataio/pgstream
+brew install pgstream
+```
+
+Or you can find pre-built binaries on the [release pages](https://github.com/xataio/pgstream/releases).
+
+Next step is to initialize `pgstream`, like this:
+
+```sh
+pgstream init --pgurl "postgres://postgres:postgres@localhost/postgres?sslmode=disable"
+```
+
+This creates several tables managed by pgstream, which are grouped in a schema called `pgstream`.
+
+## Configuration
+
+The `pgstream` repo comes with a `pg2webhook.env` configuration file for this purpose. Its contents are:
+
+```sh
+# Listener config
+PGSTREAM_POSTGRES_LISTENER_URL="postgres://postgres:postgres@localhost?sslmode=disable"
+
+# Processor config
+PGSTREAM_TRANSLATOR_STORE_POSTGRES_URL="postgres://postgres:postgres@localhost?sslmode=disable"
+PGSTREAM_WEBHOOK_SUBSCRIPTION_STORE_URL="postgres://postgres:postgres@localhost?sslmode=disable"
+PGSTREAM_WEBHOOK_SUBSCRIPTION_STORE_CACHE_ENABLED=false
+PGSTREAM_WEBHOOK_SUBSCRIPTION_STORE_CACHE_REFRESH_INTERVAL="60s"
+```
+
+In this file, there are two Postgres connection strings defined. The first one is for the listener, to which database should pgstream connect and listen for changes. The second one is used as a metadata db to store the registered subscriptions and their configuration. In this example, they are both set to the same database, but you can use different databases or instances if you prefer.
```

Comment:
Maybe we can link to the readme configuration here for more details:
https://github.com/xataio/pgstream/tree/main?tab=readme-ov-file#configuration